gammapy/background/make.py

```diff
@@ -0,0 +1,106 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import numpy as np
+from astropy.units import Quantity
+from astropy.coordinates import Angle
+from astropy.table import Table
+from astropy.io import fits
+from ..background import Cube,CubeBackgroundModel
+
+__all__ = ['make_bg_cube_model',
+           ]
+
+
+def make_bg_cube_model(observation_table, fits_path, a_la_michi=False):
+    """Create a bg model from an observation table.
+
+    Produce a background cube model using the data from an observation list.
+    Steps:
+
+    1. define binning
+    2. fill events and livetime correction in cubes
+    3. fill events in bg cube
+    4. smooth
+    5. correct for livetime and bin volume
+    6. set 0 level
+
+    TODO: review steps!!!
+
+    The background cube model contains 3 cubes: events, livetime and background.
+    The latter contains the bg cube model.
+
+    Parameters
+    ----------
+    observation_table : `~gammapy.obs.ObservationTable`
+        Observation list to use for the histogramming.
+    fits_path : str
+        Path to the data files.
+    a_la_michi : bool, optional
```

Comment:
Probably the code becomes more readable if you have one large multiple-choice section:
```python
if method == 'default':
    .... do everything ...
elif method == 'michi':
    ... do everything ...
else:
    raise ValueError()
```
instead of three small if statements as you have now.
(But I'm not 100% sure, do what you think is best.)